packages/design-system-shared/src/index.ts

```diff
@@ -7,3 +7,11 @@
 export default function greeter(name: string): string {
```

Comment:
Moved to a separate PR https://github.com/MetaMask/metamask-design-system/pull/817